opentracing-proxy/opentracing-proxy-autoconfigure/src/test/java/org/zalando/opentracing/proxy/autoconfigure/MultipleTracerTest.java

```diff
@@ -0,0 +1,57 @@
+package org.zalando.opentracing.proxy.autoconfigure;
+
+import io.opentracing.Tracer;
+import io.opentracing.mock.MockTracer;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Primary;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.zalando.opentracing.proxy.ProxyTracer;
+
+import java.util.List;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.everyItem;
+import static org.hamcrest.Matchers.hasSize;
+import static org.hamcrest.Matchers.instanceOf;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.not;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid unused imports such as 'org.hamcrest.Matchers.not'](https://app.codacy.com/app/whiskeysierra/opentracing-toolbox/pullRequest?prid=4701101)